### PR TITLE
Add mojmap option

### PR DESCRIPTION
**Pull request by Earthcomputer**
Don't merge until loom 0.5 is stable, of course :)

### CODE REVIEW

**Comment by modmuss50** [comment]
No. I don’t think we want this. Yarn is still the default and is still the only officially supported mappings. The mojang mappings aren’t something I think we should be advertising like this.

**Comment by Earthcomputer** [comment]
You still haven't been open as to why you have made that decision. Let's wait for others' opinions on this as well.

**Comment by modmuss50** [comment]
I think I’ve been open about it. Yarn IMO provides a superior experience with better names, params and documentation. The license on the mappings is still up for debate but it’s beyond me so I’ll let the people who know what they are talking about deal with that.

I’d like to hope yarn will continue on, but this will only happen if people continue to contribute to it. Right now the discussion of the mojang names is also disallowed (To protect yarn) making it hard for people to get help with these names.

File: build.gradle [hunk 1]
```diff
@@ -14,6 +14,8 @@ dependencies {
 	//to change the versions see the gradle.properties file
 	minecraft "com.mojang:minecraft:${project.minecraft_version}"
 	mappings "net.fabricmc:yarn:${project.yarn_mappings}:v2"
+	// to use Mojang's official mappings, comment the above line and uncomment below. Use at your own risk.
+	//mappings "com.mojang.minecraft:mappings:${project.minecraft_version}"
```
Comment (modmuss50) [inline build.gradle:18]: This isnt how its done either

Comment (modmuss50) [reply build.gradle:18]: `mappings minecraft.officialMojangMappings()`

Comment (Earthcomputer) [reply build.gradle]: Fixed

**Pull request closed without merging**